File: templates/matlab/reliability_maintenance_score_radar.m
function fig = reliability_maintenance_score_radar()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('radar', 3307, 'reliability and maintenance: multi-metric radar', 'reliability and maintenance', 'multi-metric radar');
end

File: templates/python/reliability_maintenance_score_radar.py
"""reliability_maintenance_score_radar: 可靠性与维修多维评分雷达（radar 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='reliability and maintenance: multi-metric radar'):
    return make_template_figure('radar', seed=3307, title=title, domain='reliability and maintenance', topic='multi-metric radar')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
